js/ui/lookup.js

```diff
@@ -233,6 +233,14 @@ const Lookup = DropDownList.inherit({
             * @hidden
             */
 
+            /**
+             * @name dxLookupOptions.disableOptionCentering
+             * @type boolean
+             * @default false @for Material
+             * @public
+             */
+            disableOptionCentering: true,
```

Comment:
Was this option name discussed with technical writers.

Reply:
The option name is 'itemCenteringEnabled' now